docker 启动镜像报错

images信息：
```
REPOSITORY                     TAG                 IMAGE ID            CREATED             SIZE
tikazyq/artipub                latest              ab81db1fcc46        7 weeks ago         2.01GB
```

docker容器日志
```
nginx: [error] open() "/run/nginx.pid" failed (2: No such file or directory)

> backend@1.0.0 start:backend /app
> node server.js

module.js:550
    throw err;
    ^

Error: Cannot find module 'axios'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/app/spiders/base.js:8:15)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/app/spiders/juejin.js:1:82)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! backend@1.0.0 start:backend: `node server.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the backend@1.0.0 start:backend script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-18T06_44_47_085Z-debug.log
nginx: [alert] kill(11, 15) failed (3: No such process)

> backend@1.0.0 start:backend /app
> node server.js

module.js:550
    throw err;
    ^

Error: Cannot find module 'axios'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/app/spiders/base.js:8:15)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/app/spiders/juejin.js:1:82)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! backend@1.0.0 start:backend: `node server.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the backend@1.0.0 start:backend script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-18T06_53_29_763Z-debug.log
nginx: [alert] kill(11, 15) failed (3: No such process)

> backend@1.0.0 start:backend /app
> node server.js

module.js:550
    throw err;
    ^

Error: Cannot find module 'axios'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/app/spiders/base.js:8:15)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/app/spiders/juejin.js:1:82)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! backend@1.0.0 start:backend: `node server.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the backend@1.0.0 start:backend script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-11-18T06_59_14_054Z-debug.log
```

> images信息：
> 
> ```
> REPOSITORY                     TAG                 IMAGE ID            CREATED             SIZE
> tikazyq/artipub                latest              ab81db1fcc46        7 weeks ago         2.01GB
> ```
> 
> docker容器日志
> 
> ```
> nginx: [error] open() "/run/nginx.pid" failed (2: No such file or directory)
> 
> > backend@1.0.0 start:backend /app
> > node server.js
> 
> module.js:550
>     throw err;
>     ^
> 
> Error: Cannot find module 'axios'
>     at Function.Module._resolveFilename (module.js:548:15)
>     at Function.Module._load (module.js:475:25)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/app/spiders/base.js:8:15)
>     at Module._compile (module.js:653:30)
>     at Object.Module._extensions..js (module.js:664:10)
>     at Module.load (module.js:566:32)
>     at tryModuleLoad (module.js:506:12)
>     at Function.Module._load (module.js:498:3)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/app/spiders/juejin.js:1:82)
>     at Module._compile (module.js:653:30)
>     at Object.Module._extensions..js (module.js:664:10)
>     at Module.load (module.js:566:32)
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! backend@1.0.0 start:backend: `node server.js`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the backend@1.0.0 start:backend script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /root/.npm/_logs/2020-11-18T06_44_47_085Z-debug.log
> nginx: [alert] kill(11, 15) failed (3: No such process)
> 
> > backend@1.0.0 start:backend /app
> > node server.js
> 
> module.js:550
>     throw err;
>     ^
> 
> Error: Cannot find module 'axios'
>     at Function.Module._resolveFilename (module.js:548:15)
>     at Function.Module._load (module.js:475:25)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/app/spiders/base.js:8:15)
>     at Module._compile (module.js:653:30)
>     at Object.Module._extensions..js (module.js:664:10)
>     at Module.load (module.js:566:32)
>     at tryModuleLoad (module.js:506:12)
>     at Function.Module._load (module.js:498:3)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/app/spiders/juejin.js:1:82)
>     at Module._compile (module.js:653:30)
>     at Object.Module._extensions..js (module.js:664:10)
>     at Module.load (module.js:566:32)
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! backend@1.0.0 start:backend: `node server.js`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the backend@1.0.0 start:backend script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /root/.npm/_logs/2020-11-18T06_53_29_763Z-debug.log
> nginx: [alert] kill(11, 15) failed (3: No such process)
> 
> > backend@1.0.0 start:backend /app
> > node server.js
> 
> module.js:550
>     throw err;
>     ^
> 
> Error: Cannot find module 'axios'
>     at Function.Module._resolveFilename (module.js:548:15)
>     at Function.Module._load (module.js:475:25)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/app/spiders/base.js:8:15)
>     at Module._compile (module.js:653:30)
>     at Object.Module._extensions..js (module.js:664:10)
>     at Module.load (module.js:566:32)
>     at tryModuleLoad (module.js:506:12)
>     at Function.Module._load (module.js:498:3)
>     at Module.require (module.js:597:17)
>     at require (internal/module.js:11:18)
>     at Object.<anonymous> (/app/spiders/juejin.js:1:82)
>     at Module._compile (module.js:653:30)
>     at Object.Module._extensions..js (module.js:664:10)
>     at Module.load (module.js:566:32)
> npm ERR! code ELIFECYCLE
> npm ERR! errno 1
> npm ERR! backend@1.0.0 start:backend: `node server.js`
> npm ERR! Exit status 1
> npm ERR! 
> npm ERR! Failed at the backend@1.0.0 start:backend script.
> npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /root/.npm/_logs/2020-11-18T06_59_14_054Z-debug.log
> ```

我也同样错误，怎么解决的？

--- a/backend/package.json
+++ b/backend/package.json
@@ -11,6 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "async-lock": "^1.2.2",
+    "axios": "^0.20.0",
     "body-parser": "^1.19.0",
     "clipboardy": "^2.1.0",
     "cors": "^2.8.5",

